package.json

```diff
@@ -6,8 +6,11 @@
   "scripts": {
     "test": "jest --coverage",
     "test:watch": "npm run test -- --watch -o --coverage",
+    "test:changed-feature-branch": "jest --changedSince=github.head_ref --coverage --passWithNoTests",
```

Comment:
This is failing in the action as the value for `--changedSince` needs to be passed via the action like

```npm run test -- --changedSince=${{ github.head_ref }}```